app/boards/shields/cradio/cradio.zmk.yml

```
@@ -0,0 +1,12 @@
+file_format: "1"
+id: cradio
+name: Cradio/Ferris Sweep
```

At some point, [Sweep](https://github.com/davidphilipbarr/Sweep) dropped "Ferris" from the name to avoid potential confusion with the original [Ferris](https://github.com/pierrechevalier83/ferris). It's now referred to simply as Sweep in its documentation: https://github.com/davidphilipbarr/Sweep.

For consistency, and given that original Ferris will also be in ZMK someday with https://github.com/zmkfirmware/zmk/pull/642, would it make sense to go with simply "Sweep" here also?

```suggestion
name: Cradio/Sweep
```

---

Good catch. Fixed.